apps/styleguide/src/forms/InputOntology.vue

```diff
@@ -283,22 +287,32 @@ export default {
         });
       } else {
         //non-list, deselect all
-        this.terms.forEach((term) => (term.selected = false));
+        Object.keys(this.terms).forEach(
+          (key) => (this.terms[key].selected = false)
+        );
+      }
```

Comment:
code duplication , can be moved outside the if

Reply:
the duplicate is actually in another method

Reply:
I don't think so? the duplicate is in another function 'select'